packages/tools/kadena-cli/src/tx/utils/txHelpers.ts

```diff
@@ -679,3 +680,10 @@ export async function logTransactionDetails(command: ICommand): Promise<void> {
     log.info(`No transaction details to display`);
   }
 }
+
+export const getTxTemplateDirectory = (): string | null => {
+  const kadenaDir = services.config.getDirectory();
+  return isNotEmptyString(kadenaDir)
```

Comment:
why did you decide to use `isNotEmptyString` everywhere on `config.getDirectory()`'s response? It's type is `string | null` so just checking non-null should be sufficient. 

I understand this works fine, but the wording of the method, and checking for something (empty string) that it can never be will cause confusion for code readability.